packages/desktop-electron/menu.ts

```diff
@@ -173,7 +174,16 @@ export function getMenu(
       role: 'help',
       submenu: [
         {
-          label: 'Keyboard Shortcuts Reference',
+          label: 'Documentation',
+          async click(_menuItem, focusedWin) {
+            const page = await focusedWin?.webContents.executeJavaScript(
+              'window.location.pathname',
```

Comment:
Open to feedback if there's a more standard way to do this, not too familiar with Electron